calculator.js

```diff
@@ -1,9 +1,33 @@
 "use strict";
 
 export class Calculator {
+  constructor(x){
+    this._x = x;
+  }
   //write your code here
+  addition(y) {
+      return this._x+y;
```

Comment:
butuh di revisi sedikit lagi, agar method chaining nya jalan.
yg di return adalah object nya yaitu this, sehingga bisa dipanggil dengan cara : 
console.log(Advanced.addition(10).substraction(2));